README.md

```diff
@@ -0,0 +1,169 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>Rails Recipe App</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Rails Budget App ](#-rails-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails Budget App <a name="about-project"></a>
+
+**Budget app** An appliation help to track users expenses. The app has authunication for security reasons.
+
+<a href="https://www.loom.com/share/a1e6cae6108346f9a3e74d6c53418c0f?sid=3c9ddcfe-e7ab-4f19-9488-f871d2b4db1a">Presentation vedio about the project</a>
+
```

Comment:
- [ ] Your app is apparently not deployed, as live demo link cannot be found in the ReadMe. Kindly ensure to/that:
  * deploys the application using Render, Heroku or similar for Rails app.
  * a link to the deployed project is present in the ReadMe.